src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -93,6 +95,12 @@ private CloudSdk(Path sdkPath, String appCommandMetricsEnvironment,
     this.processRunner = new DefaultProcessRunner(async, stdOutLineListeners, stdErrLineListeners,
         exitListeners, startListeners);
 
+    // Populate jar locations.
```

Comment:
In the library we can't assume there's a user. Any asking of user needs to be done from the UI.